x-pack/legacy/plugins/snapshot_restore/__jest__/client_integration/home.test.ts

```diff
@@ -458,7 +457,7 @@ describe.skip('<SnapshotRestoreHome />', () => {
           expect(row).toEqual([
             snapshot.snapshot, // Snapshot
             REPOSITORY_NAME, // Repository
-            formatDate(snapshot.startTimeInMillis), // Date created
+            'foo', // TODO: fix this with FormattedDateTime value
```

Comment:
Did you intend to address these TODOs as part of this PR?

Reply:
no, these are skipped tests that need to be updated overall in a different PR, this TODO is just a reminder